irohad/model/execution/impl/command_executor.cpp

```diff
@@ -164,14 +161,11 @@ namespace iroha {
         const std::string &creator_account_id) {
       auto cmd_value = static_cast<const CreateRole &>(command);
 
-      if (not commands.insertRole(cmd_value.role_name)) {
-        return makeExecutionError("failed to insert role: "
-                                  + cmd_value.role_name);
-      }
-
-      return errorIfNot(commands.insertRolePermissions(cmd_value.role_name,
-                                                       cmd_value.permissions),
-                        "failed to insert role permissions");
+      auto result = commands.insertRole(cmd_value.role_name) | [&]() {
```

Comment:
You may omit `()`

Reply:
Yes, it is done in wsv_command, seems like i forgot to do that here